main.go

```diff
@@ -75,5 +91,10 @@ func main() {
 
 func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+
+	flag.StringVar(&etcdURL, "etcd-core-url", "", "The address of the core etcd server. Required.")
+	flag.StringVar(&etcdCAConfigMapName, "etcd-ca-name", "etcd-coreserving-ca", "The name of the ConfigMap where CA is stored.")
```

Comment:
etcd-ca-name is too unspecific IMO. Better, core-etcd-ca-configmap-name ?

Reply:
etcd-core-ca-configmap as flag name would be more consistent

Reply:
Done. This could be changed again/revised in #14.